projects/cds/src/profiletag/services/spartacus-event.service.ts

```diff
@@ -42,14 +42,12 @@ export class SpartacusEventService {
   /**
    * Listens to the changes to the cart and pushes the event for profiletag to pick it up further.
    */
-  cartChanged(): Observable<{ entries: OrderEntry[]; cart: Cart }> {
-    return combineLatest([
-      this.cartService.getEntries(),
-      this.cartService.getActive(),
-    ]).pipe(
-      skipWhile(([entries]) => entries.length === 0),
-      map(([entries, cart]) => ({
-        entries,
+  cartChanged(): Observable<{ cart: Cart }> {
+    return this.cartService.getActive().pipe(
+      skipWhile(
+        cart => typeof cart.entries === 'undefined' || cart.entries.length === 0
```

Comment:
I prefer !Boolean(cart.entries), what do you think?

Reply:
Updated